### ISSUE TITLE
Uncaught TypeError: can't access property "registerPlugin", YoastSEO.app is undefined

### ISSUE THREAD

**Issue by jasperf** [soflyy/oxygen-bugs-and-features#1601]
**Describe the bug**
Saving products does not work well and hangs sometimes. When I check the console log I see 

```
Uncaught TypeError: can't access property "registerPlugin", YoastSEO.app is undefined 
```

or

```
Uncaught TypeError: can't access property "registerPlugin", YoastSEO.app is undefined
    YoastOxygen https://staging.site.com/wp-content/plugins/wpdevdesign-oxygen-yoast-seo-master//assets/js/yoast-seo-compatibility.js?ver=5.5.1:4
    <anonymous> https://staging.site.com/wp-content/plugins/wpdevdesign-oxygen-yoast-seo-master//assets/js/yoast-seo-compatibility.js?ver=5.5.1:20
    jQuery 4
yoast-seo-compatibility.js:4:4
    YoastOxygen https://staging.site.com/wp-content/plugins/wpdevdesign-oxygen-yoast-seo-master//assets/js/yoast-seo-compatibility.js?ver=5.5.1:4
    <anonymous> https://staging.site.com/wp-content/plugins/wpdevdesign-oxygen-yoast-seo-master//assets/js/yoast-seo-compatibility.js?ver=5.5.1:20
    jQuery 4
Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
    in select (created by y)
    in div (created by u)
    in u (created by y)
    in y (created by m)
    in m (created by y)
    in div (created by Context.Consumer)
    in t (created by SchemaTab__SchemaContainer)
    in SchemaTab__SchemaContainer (created by y)
    in y (created by Context.Consumer)
    in f (created by WithDispatch(f))
    in WithDispatch(f)
    in Unknown (created by WithSelect(WithDispatch(f)))
    in WithSelect(WithDispatch(f)) (created by w)
    in o (created by s)
    in t (created by s)
    in s (created by w)
    in div (created by o)
    in o (created by w)
    in SlotComponent (created by Context.Consumer)
    in Slot (created by slot_fill_Slot)
    in slot_fill_Slot
    in Unknown (created by l)
    in a (created by l)
    in l
    in slot_fill_provider_SlotFillProvider (created by SlotFillProvider)
    in SlotFillProvider react-dom.js:3412:15

```
**To Reproduce**
1. Go to 'WooCommerce Products'
2. Click on 'any product to edit'
3. Save product
4. See Yoast SEO Error

**Expected behavior**
A product save without issues and with Yoast SEO working

**What actually happens**
Things did not save well and I was also logged off for no reason once

**Screenshots**
*If applicable, add screenshots to help explain your problem.*

**Desktop (please complete the following information):**
- OS: [e.g. macOS]
 - Browser [FireFoX]
 - Oxygen Version [3.5]

**Comment by jasperf** [comment]
Seems to be an issue with WP DevDesign's plugin I realized now so closing this issue for now.

**Comment by jasperf** [comment]
Hmm even when turned off I do still see

```
Uncaught TypeError: can't access property "registerPlugin", YoastSEO.app is undefined
    YoastOxygen https://staging.site.com/wp-content/plugins/oxygen/component-framework/admin/js/yoast-seo-compatibility.js?ver=5.5.1:4
    <anonymous> https://staging.site.com/wp-content/plugins/oxygen/component-framework/admin/js/yoast-seo-compatibility.js?ver=5.5.1:20
    jQuery 4
yoast-seo-compatibility.js:4:4
```

**Comment by KittenCodes** [comment]
@jasperf Yoast made a change in 14.9 which broke Oxygen's Yoast integration. You can either downgrade to Yoast 14.8, or wait for Oxygen 3.6 to be released where we've fixed the issue.

**Comment by wpsumo** [comment]
@KittenCodes Neither this one is fixed for me:
```
Uncaught TypeError: Cannot read property 'registerPlugin' of undefined
    at new YoastOxygen (yoast-seo-compatibility.js?ver=5.5.3:4)
    at HTMLDocument.<anonymous> (yoast-seo-compatibility.js?ver=5.5.3:20)
    at i (jquery.js?ver=1.12.4-wp:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4-wp:2)
    at Function.ready (jquery.js?ver=1.12.4-wp:2)
    at HTMLDocument.J (jquery.js?ver=1.12.4-wp:2)
```

**Comment by KittenCodes** [comment]
@wpsumo It sounds as though there's some caching on your site that is preventing you from seeing the fixes added to v3.6.
